lua/tests/automated/pickers/find_files_spec.lua

```diff
@@ -107,12 +111,12 @@ describe("builtin.find_files", function()
 
   it("should be able to c-n the items", function()
     tester.run_string [[
-      runner.picker('find_files', 'fixtures/file<c-n>', {
+      runner.picker('find_files', 'fixtures/find_files/file<c-n>', {
```

Comment:
I have included added a subdirectory alongside of fixing this test to prepare for other fixtures in my next PR.

Reply:
makes sense. thanks :)